### ISSUE TITLE
Flux networks crashes game

### ISSUE THREAD

**Issue by Fireslar** [FTBTeam/FTB-Modpack-Issues#6021]
### Modpack

FTB Interactions Remastered

### Modpack version

1.7.0

### Launcher

FTB App

### Has the pack been modified

No

### Log Files

[crash-2024-10-28_01.55.10-client.txt](https://github.com/user-attachments/files/17537870/crash-2024-10-28_01.55.10-client.txt)

### Describe the bug

When assessing Flux networks interface there is an almost 100% chance that the game freezes and crashes.

### Steps to reproduce

1. place down flux point/plug
2. right click to open interface
3. click through tabs
4. game freezes 

### Expected behaviour

Be allowed into interface without game freezing

**Comment by Gaz492** [comment]
Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod (https://github.com/SonarSonic/Flux-Networks/issues/455)

I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do.
Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?

**Comment by Fireslar** [comment]
> Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod ([SonarSonic/Flux-Networks#455](https://github.com/SonarSonic/Flux-Networks/issues/455))
> 
> I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do. Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?

I have placed it on cobblestone, GT cables, hatches and converters, sometimes it works but usually when I try to set it to a network it freezes

**Comment by Fireslar** [comment]
> Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod ([SonarSonic/Flux-Networks#455](https://github.com/SonarSonic/Flux-Networks/issues/455))
> 
> I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do. Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?

Also, this might be a fluke but, I was just able to use it fine on my space station with just one plug/point, a orbital laser, warp core/drive and a AS teleporter, but when I tried back in the void where all my stuff is it crashed.

**Comment by Fireslar** [comment]
> > > Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod ([SonarSonic/Flux-Networks#455](https://github.com/SonarSonic/Flux-Networks/issues/455))
> > > I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do. Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?
> > 
> > 
> > Also, this might be a fluke but, I was just able to use it fine on my space station with just one plug/point, a orbital laser, warp core/drive and a AS teleporter, but when I tried back in the void where all my stuff is it crashed.
> 
> Would you be able to send a copy of your world so we can try replicating this? Unfortunately, I'm still not able to replicate the issue, which indicate that there is a specific setup/configuration that's causing the issue.

Sure, do you want the whole instance folder or just the save folder, also where would I send it?

**Comment by Gaz492** [comment]
> > > > Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod ([SonarSonic/Flux-Networks#455](https://github.com/SonarSonic/Flux-Networks/issues/455))
> > > > I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do. Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?
> > > 
> > > 
> > > Also, this might be a fluke but, I was just able to use it fine on my space station with just one plug/point, a orbital laser, warp core/drive and a AS teleporter, but when I tried back in the void where all my stuff is it crashed.
> > 
> > 
> > Would you be able to send a copy of your world so we can try replicating this? Unfortunately, I'm still not able to replicate the issue, which indicate that there is a specific setup/configuration that's causing the issue.
> 
> Sure, do you want the whole instance folder or just the save folder, also where would I send it?

Just the save will do, a google drive/onedrive link or anything like that will be fine

**Comment by Fireslar** [comment]
> > > > > Unfortunately, I'm not sure there's anything we can do to resolve this issue as the mod author no supports this version of the mod ([SonarSonic/Flux-Networks#455](https://github.com/SonarSonic/Flux-Networks/issues/455))
> > > > > I was unable to reproduce the issue either which makes it a little harder to see if there is something we can do. Did you place the flux point/plug down on anything specific to cause this, or do you have any specific item in your inventory when this happens?
> > > > 
> > > > 
> > > > Also, this might be a fluke but, I was just able to use it fine on my space station with just one plug/point, a orbital laser, warp core/drive and a AS teleporter, but when I tried back in the void where all my stuff is it crashed.
> > > 
> > > 
> > > Would you be able to send a copy of your world so we can try replicating this? Unfortunately, I'm still not able to replicate the issue, which indicate that there is a specific setup/configuration that's causing the issue.
> > 
> > 
> > Sure, do you want the whole instance folder or just the save folder, also where would I send it?
> 
> Just the save will do, a google drive/onedrive link or anything like that will be fine
https://drive.google.com/drive/folders/1hDWDgOMhbOvqhhpDH5fGuu6AOXROizvn?usp=drive_link

**Comment by ArtpokeBr** [comment]
I'll wait until you confirm the exact crash here @Gaz492 =)

**Comment by Delfayne** [comment]
FWIW, I've used Flux Networks extensively throughout my world with zero crash problems so far.

**Comment by yuuki1293** [comment]
I used to have the same problem (ver 1.3.0), but it stopped crashing at some point. At that time I joined the server and when I switch tabs it freezes without crashing.

**Comment by Fireslar** [comment]
> I used to have the same problem (ver 1.3.0), but it stopped crashing at some point. At that time I joined the server and when I switch tabs it freezes without crashing.

Yeah, when I switch tabs I can see the game quickly slow and if I quickly close out of the interface sometimes it won't freeze but normally It freezes and I have to close the game.

**Comment by Gaz492** [comment]
So I was able to reproduce the freezes that you both mentioned, but the freezes only lasted for at most a second, and didn't cause the game to crash.

@ArtpokeBr are you able to reproduce this?

**Comment by ArtpokeBr** [comment]
I actually wasn't able to properly reproduce this

**Comment by Fireslar** [comment]
It doesn't immediately crash, it freezes and when I click the game stops responding and I have to close the game
